jupyterhub/templates/proxy/autohttps/deployment.yaml

```diff
@@ -6,31 +6,19 @@ kind: Deployment
 metadata:
```

Comment:
Should this be a deployment or statefulset? A deployment will do a rolling update and have two replicas running at the same time during updates while a statefulset would shut down and then start back up.

Reply:
I think deployment is the right call here - there isn't a lot of state here, and deployment will make sure we aren't dropping user requests during deployment.